go.mod

```diff
@@ -10,28 +10,28 @@ require (
 	github.com/hashicorp/terraform-plugin-docs v0.13.0
 	github.com/hashicorp/terraform-plugin-go v0.14.3
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.24.1
-	github.com/indykite/jarvis-sdk-go v0.2.3
-	github.com/onsi/ginkgo/v2 v2.7.0
-	github.com/onsi/gomega v1.24.2
-	google.golang.org/grpc v1.51.0
-	google.golang.org/protobuf v1.28.1
+	github.com/indykite/jarvis-sdk-go v0.5.0
+	github.com/onsi/ginkgo/v2 v2.7.1
+	github.com/onsi/gomega v1.26.0
+	google.golang.org/grpc v1.52.3
+	google.golang.org/protobuf v1.28.2-0.20220831092852-f930b1dc76e8
```

Comment:
It this some "alpha" of `1.28.2`? Even on https://google.golang.org/protobuf I see only version `1.28.1`

Reply:
It's the next master to be released. Some other dependency was require this but it may be released before we release.